brayns/io/NESTLoader.h

```diff
@@ -51,11 +51,22 @@ namespace brayns
  * forward.
  * @todo Move this loaded to Brion
  */
-class NESTLoader : public ProgressReporter
+class NESTLoader : public Loader
 {
 public:
     NESTLoader(const GeometryParameters& geometryParameters);
 
+    void importFromBlob(Blob&&, Scene&, const Matrix4f&, const size_t) final
+    {
+        throw std::runtime_error("Unsupported");
+    }
+
+    void importFromFile(const std::string&, Scene&, const Matrix4f&,
+                        const size_t) final
+    {
+        throw std::runtime_error("Unsupported");
```

Comment:
this class also can't load anything? Why is it not calling importCircuit() / importSpikeReport()?

Reply:
exactly, which one to call? The unified load API does not work for some of those 'legacy' loaders. Not even sure if we need that. Hence, for now those loaders are not supported by the 'new' way of loading data. The 'legacy' way of loading is not affected by that, though.

Reply:
But I don't understand what is the blocking factor, is it just lack of time to do it in this PR? If the problem is choosing between 2 methods just make 2 Loader classes, no..?

Reply:
I'm not spending time on something we don't use atm. I leave it to @favreau to either adapt this loader or remove it.

Reply:
I would just remove it for now. This will have to go to a plug-in in any case. Don't waste time on that loader.